src/test/java/firstSimpleJavaTasks/MaxMinAverageTests.java

```diff
@@ -0,0 +1,48 @@
+package firstSimpleJavaTasks;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class MaxMinAverageTests {
+
+    private int[] newArray = {0,1,2,3,4,5,6,7,8,9};
+    private int[] zeroLengthArray = {};
+    private int[] arrayWithNegativeValues = {-5,225,-1,0,25,-61};
+
+
+    @Test
+    public void testGettingMaxValue() {
+        int n = MaxMinAverage.getMaxIntArrayValue(newArray);
```

Comment:
you can use single-line asserts in such cases without creating a temporary variable.

Reply:
Let's discuss this on our daily meeting. As for me, form with temp variable looks more friendly than "single-line". Below is 2 examples for visibility.

@Test
    public void testGettingMaxValue() {
        int n = MaxMinAverage.getMaxIntArrayValue(newArray);
        Assert.assertEquals(9, n);
    }

    @Test
    public void testGettingMaxValueArrayWithNegativeValues() {
        Assert.assertEquals(225, MaxMinAverage.getMaxIntArrayValue(arrayWithNegativeValues));
    }